mbed_flasher/flashers/FlasherMbed.py

```diff
@@ -199,17 +202,34 @@ def copy_file(self, source, destination):
 
         self.logger.debug("SHA1: %s", hashlib.sha1(aux_source).hexdigest())
 
-        self.logger.debug("writing binary: %s (size=%i bytes)", destination, len(aux_source))
         try:
-            with open(destination, "wb", 0) as new_file:
-                new_file.write(aux_source)
-                new_file.flush()
-                os.fsync(new_file.fileno())
-        except (IOError, OSError):
+            if platform.system() == "Windows":
+                self._copy_file_windows(source, destination)
+            else:
+                self._copy_file(aux_source, destination)
+        except (IOError, OSError, subprocess.CalledProcessError):
             self.logger.exception("File couldn't be copied")
             raise FlashError(message="File couldn't be copied",
                              return_code=EXIT_CODE_OS_ERROR)
 
+    def _copy_file_windows(self, source, destination):
+        command = "copy \"{}\" \"{}\"".format(os.path.abspath(source), destination)
+        self.logger.debug("Copying with command: {}".format(command))
+        subprocess.check_call(command)
```

Comment:
this is still an problem, need some change

Reply:
Why is it a problem?

Reply:
I don't understand the purpose to start a new command interpreter, would mind explain a bit?
```
C:\>cmd /?
Starts a new instance of the Windows XP command interpreter
```

Reply:
It is one way to avoid using shell=True, which we do not want to do for security reasons.